app/models/jobsModel.js

```diff
@@ -1,30 +1,26 @@
 const db = require('../database/dbConfig.js')
 
 /**
- * Inserts a job into the database.
- * scheduled_time is an optional field.
- * 
- * @param {"class_id" : integer, 
- *         "scheduled_time" : Date,
- *         "job_hash" : string} job 
+ * Inserts a job into the database. scheduled_time is an optional field.
+ *
+ * @param job
+ * @returns {Promise<void>}
```

Comment:
Why is the return type {Promise void} if the function doesn't return anything? Is this different from just saying @return none or @return void?

Reply:
No, this is the same, interesting thing is my ide generates jsDocs for me, and since this is an async function, it just assumes it returns a promise, but yeah, Promise<void> means it returns nothing.